HandleNHNotificationCall/handler.ts

```diff
@@ -11,6 +11,7 @@ import { KindEnum as DeleteKind } from "../generated/notifications/DeleteInstall
 import { KindEnum as NotifyKind } from "../generated/notifications/NotifyMessage";
 
 import { OrchestratorName as CreateOrUpdateInstallationOrchestrator } from "../HandleNHCreateOrUpdateInstallationCallOrchestrator/handler";
+import { OrchestratorName as DeleteInstallationOrchestratorName } from "../HandleNHDeleteInstallationCallOrchestrator/handler";

```

Comment:
Question: how is this different than 
```ts
const DeleteInstallationOrchestratorName = "HandleNHDeleteInstallationCallOrchestrator";
```
?

Reply:
We get it from the `HandleNHNotifyMessageCallOrchestrator` itself, so if we want to change the Orchestrator name, we'll just need to take care of it into the Orchestrator folder, we won't anymore need to do a search and replace for const strings.

Reply:
Sorry, I was too tricky 😅 

I mean: the orchestrator name is defined by it's folder; we reference such folder to get a variable which contains the name of the folder we just referenced 🙃 .
Best scenario: we could have save one `import`.
Worst scenario: the variable exposes a misspelled value.

Reply:
I don't get it.. I mean, it's true that in both scenarios in case of renaming we need to be careful with renaming things. 
But I think it's a matter of who is in charge of changing it. 
In future we can try to extract it directly from the belonging directory like what you said for the activity name..